test/t/test_arp.py

```diff
@@ -5,7 +5,7 @@
 
 class TestArp:
     @pytest.mark.xfail(in_docker(), reason="Probably fails in docker")
-    @pytest.mark.complete("arp ")
+    @pytest.mark.complete("arp ", skipif='test -z "$(arp 2>/dev/null)"')
```

Comment:
I think the docker xfail and related in_docker import can be removed, this seems a better and more general fix?

Reply:
Sure. I wasn't sure what was the problem with docker.

Reply:
I'm fairly confident that it was this one. We probably have some other ones that could be skipped more cleanly than doing a container check, too (nowadays grep for `in_container` if interested).